packages/vite/src/node/plugins/reporter.ts

```diff
@@ -301,7 +301,7 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
     },
 
     closeBundle() {
-      if (shouldLogInfo && !config.build.watch) {
```

Comment:
If the build failed, `buildEnd` hook has the first parameter. I think it will be cleaner if we used that instead of `buildBreakInfo`.
https://rollupjs.org/plugin-development/#buildend

Reply:
Thank you for your suggestion, modifications have been made.